packages/juniper_srx/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.8.1"
```

@ernst-s Could you bump the minor instead of patch version? Also change the `manifest.yml`
```suggestion
- version: "1.9.0"
```

---

Done